.circleci/config.yml

```diff
@@ -331,9 +331,20 @@ jobs:
           docker login -u $DOCKER_USER -p $DOCKER_PASS
           docker push tendermint/gaia:$GAIAD_VERSION
 
-          # Push the tag also for the version, so clients can build up on top of it
-          GAIAD_VERSION=`/tmp/workspace/bin/gaiad version`
-          docker push tendermint/gaia:$GAIAD_VERSION
```

Comment:
I removed the hashed version, in favour of the git tag version that will run only when a new tag will be added

Reply:
Why not using git describe's output?

Reply:
It seemed to me more readable to use $CIRCLE_TAG instead of getting the tag from a command.
In here the specific reason to change the behaviour was also to trigger the new job only when the tag is pushed.

Reply:
> It seemed to me more readable to use $CIRCLE_TAG instead of getting the tag from a command.

The risk is that versions may diverge. Did you check that `gaiad version` kinda looks like the image's actual version?